runtimes/java/select-java-version.md

```diff
@@ -7,6 +7,10 @@ tags:
 
 ## Selecting your java version
 
-Simply create a file **clevercloud/java_version** containing just the java version you want inside.
+Simply set the environment variable **JAVA_VERSION** to the version you want
 
 Accepted values are `6`, `7` and `8`
+
+## Retrocompatibility with the old configuration system
+
+If you have set your java version with the old system (wich consist by set the version in a file **clevercloud/java_version**), don't worry, we guarantee retrocompatibility with this configuration
```

Comment:
Maybe you should get ride of the title and just add:

"As for the applications created before <month/day/year the update happened>, their JAVA_VERSION configuration can still be made through the **clevercloud/java_version** file. This is backward compatible."

And I think you should add a note about contacting the support if the user wants to migrate from the old way to the new, giving its appID.